### PR TITLE
put link to privacy policy in the text rather than separate

### PR DESCRIPTION
**Pull request by EmmaFrith**
@ethax-ross - I'm guessing I've done something funky here!? was unable to review as I made the change

### CODE REVIEW

**Comment by MylesJarvis** [comment]
image

One for @ethax-ross to explain, but my interpretation of the code is that it's set up like it is because we link to a specific privacy policy ID stored in the CRM and then pull that for the private policy page.

**Comment by MylesJarvis** [comment]
And it would need to be coded in html to link effectively

**Comment by ethax-ross** [comment]
I've pushed a commit to fix the link @EmmaFrith, it wasn't working the way you had it because the TTA service doesn't support markdown like the GiT website does I'm afraid

**Review by peteryates — DISMISSED** [review]

File: app/views/teacher_training_adviser/steps/_accept_privacy_policy.html.erb [hunk 1]
```diff
@@ -1,11 +1,7 @@
 <% policy_id = session[:privacy_policy_id].presence || GetIntoTeachingApiClient::PrivacyPoliciesApi.new.get_latest_privacy_policy.id %>
 
 <%= f.govuk_check_boxes_fieldset :accepted_policy_id, legend: { text: "Read and accept the privacy policy"} do %>
-  <p>Our privacy notice explains how we use your personal data. It is important that you have read this notice before continuing.</p>
-
-  <div class="govuk-form-group">
-    <%= link_to "Privacy policy", privacy_policy_path(:id => policy_id) %>
-  </div>
+  <p>Read our <%= link_to("privacy policy", privacy_policy_path, target: "blank") %> before you continue. It explains how we use your personal data.</p>
```
Comment (MylesJarvis) [inline app/views/teacher_training_adviser/steps/_accept_privacy_policy.html.erb:4]: @ethax-ross I note the code in the removals also had a reference to `policy_id`, do we need to reference that here or not?

Comment (ethax-ross) [reply app/views/teacher_training_adviser/steps/_accept_privacy_policy.html.erb]: Ah, yes well spotted - that ensures the policy they accept is the one that we have shown them (on the off chance it changes after they view it but before they submit the form). Updated.

**Review by peteryates — APPROVED** [review]

**Pull request merged**